infra.bs

```diff
@@ -1426,6 +1426,28 @@ out-of-bounds, except when used with <a for=list>exists</a>.
 "<code>b</code>", "<code>c</code>", "<code>a</code>" ». Then |example|[1] is the <a>string</a>
 "<code>b</code>".
 
+<p>For notational convenience, a multiple assignment syntax may be use to assign multiple variables
+to the <a>list</a>'s <a for=list>items</a>, by surrounding the variables to be assigned by « »
+characters and separating each variable name with a comma. To
+<dfn export for=list>assign multiple variables</dfn> from a <a>list</a> is to <a>Assert</a> that the
```

Comment:
I don't think we need a definition as we'd never reference it.

I also don't like the way this "Assert" is inline and with an uppercase "a". Hmm. I think as above ("The index cannot be ...") we can state it as a requirement instead.